examples/basics/plotting/plot.py

```diff
@@ -11,13 +11,13 @@
 
 fig = vp.Fig(size=(600, 500), show=False)
 
-# Plot the target square wave shape 
+# Plot the target square wave shape
```

Comment:
hm, nope. I think it's because there is no padding on the left side - I'll add some.